implementations/EWF.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "EWF",
+  "implementation": "Energy Web Foundation VC-API",
+  "issuers": [{
+    "id": "did:key:z6MksoRPRqnMWiivV4weRGGS9SiHXuqfJEYu95EiYtbvBxW6",
+    "endpoint": "https://vc-api-dev.energyweb.org/v1/vc-api/credentials/issue",
+    "tags": ["vc-api", "Ed25519Signature2020"]
+
+  }],
+  "verifiers": [{
+    "id": "",
+    "endpoint": "https://bridge.learncard.com/credentials/verify",
```

Comment:
@artursudnik I just noticed this. Is this value correct?

Reply:
Oh, no! This is my mistake when copy-pasting. I am fixing this.

Reply:
https://github.com/w3c-ccg/vc-api-test-suite-implementations/pull/53